react-data-converter/client/package-lock.json

```diff
@@ -0,0 +1,5990 @@
+{
```

Comment:
Do we need package.json as well ?

Reply:
I think both `package.json` and `package-lock.json` are needed. 

- `package.json` holds what dependencies to be installed, and currently defines a proxy for the backend
- `package-lock.json` holds the versions of each package, so no dependency issues occur if another person wants to contribute to the webpage